README.md

```diff
@@ -70,17 +97,20 @@ For this setup to work you need to have a Docker service installed. You can get
 Create `TimeSeries` object from a Pandas DataFrame, and split in train/validation series:
```

Comment:
I'd move the demo section very up in the README.

Reply:
+1